src/payment/strategies/moneris/moneris.ts

```diff
@@ -6,6 +6,22 @@ export default interface MonerisStylingProps {
     csstexboxCvd?: string;
 }
 
+export interface MoneriesHostedFieldsQueryParams {
+    id: string;
+    pmmsg: boolean;
+    css_body: string;
+    css_textbox: string;
+    css_textbox_pan: string;
+    enable_exp: number;
+    css_textbox_exp: string;
+    enable_cvd: number;
+    css_textbox_cvd: string;
+    display_labels: number;
+    pan_label?: string;
+    exp_label?: string;
+    cvd_label?: string;
```

Comment:
These are actually not optional in `MoneriesHostedFieldsQueryParams`. Now that we've defined a type just for the query params, there's no need to use `Required`, just make them mandatory here.